### DISCUSSION TITLE
Increase area of being able to right click scroll

### DISCUSSION THREAD

**Discussion by 19peytonsmith** [ppy/osu#19492] [Ideas]
Turned on the option to right click scroll and tried it but it didn't work at first, turns out I had to place the cursor closer to the right. But yeah, I think increasing this area more to the left would be ideal and more of how it feels on stable, a nice quality of life change really :)

**Comment by frenzibyte** [comment]
See https://github.com/ppy/osu/issues/2238#issuecomment-657200248, there has been a long issue thread about it, with no simple solution to it.
